Tests/StripeTests/StripeTests.swift

```diff
@@ -189,7 +160,7 @@ final class StripeTests: XCTestCase {
 }
 """
 
-    let plan = try JSONDecoder().decode(Plan.self, from: Data(jsonString.utf8))
+    let plan = try Stripe.jsonDecoder.decode(Plan.self, from: Data(jsonString.utf8))
```

Comment:
Using the snake-case coding strategies found some tests that weren't using the right decoders.